src/test/javascript/spec/service/grading-system.service.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { getTestBed, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { GradingSystemService } from 'app/grading-system/grading-system.service';
+import { GradingScale } from 'app/entities/grading-scale.model';
+import { take } from 'rxjs/operators';
+import { RouterTestingModule } from '@angular/router/testing';
+
+describe('Grading System Service', () => {
+    let injector: TestBed;
+    let service: GradingSystemService;
+    let httpMock: HttpTestingController;
+    let elemDefault: GradingScale;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [HttpClientTestingModule, RouterTestingModule],
+        });
+        injector = getTestBed();
+        service = injector.get(GradingSystemService);
+        httpMock = injector.get(HttpTestingController);
+
+        elemDefault = new GradingScale();
+    });
+
+    afterEach(() => {
+        httpMock.verify();
+    });
+
+    it('should find a grading scale for course', async () => {
+        const returnedFromService = Object.assign(
+            {
+                id: 123,
+            },
+            elemDefault,
+        );
+        service.findGradingScaleForCourse(123).pipe(take(1)).subscribe();
+
+        const request = httpMock.expectOne({ method: 'GET' });
+        request.flush(JSON.stringify(returnedFromService));
+    });
+
+    it('should find a grading scale for exam', async () => {
+        const returnedFromService = Object.assign(
+            {
+                id: 456,
+            },
+            elemDefault,
+        );
+        service.findGradingScaleForExam(123, 456).pipe(take(1)).subscribe();
```

Comment:
If service method is returning something you should test it, also again please follow the client documentation for all of these service tests.